### PR TITLE
Templates: Added links to DIY, Fashion, and Fitness

### LINKED ISSUE #2888: Add links to product anchors in all templates
Please add dummy links back into templates.

### PR DESCRIPTION
**Pull request by mariano-formidable**
## Summary

This PR adds links to the link pages for our templates.

Here's a list of our templates and which page was updated:
- `Cooking`: This template does _not_ have a links page, no update required.

- `Entertainmnet`: This template does _not_ have a links page, no update required.

- `Travel`:  This template does _not_ have a links page, no update required.

- `Beauty`: Links will be added to its links page in a _separate_ PR.

- `Wellbeing`: Links were added to this template in [another PR](https://github.com/google/web-stories-wp/pull/3018), so no changes were made in this PR.

- `DIY`: Links were added to page 9.

- `Fashion`: Links were added to page 8.

- `Fitness`: Links were added to page 7.

## To-do

- The Beauty template will get updated in a separate PR.

## User-facing changes

- The pages list above should have links embedded in them that link to `example.com`.

## Testing Instructions

1.) Go to "Explore Templates".
2.) Click on "Use Template" to create 3 new stories from the `DIY`, `Fashion`, and `Fitness` templates.
3.) Click on "Open in Editor" for each new story and navigate to its links page (DIY - page 9, Fashion - page 8, Fitness - page 7) and hover over the center of each link to see "Example Domain" pop up for each one.

Fixes #2888

### CODE REVIEW

**Review by carloskelly13 — APPROVED** [review]

**Review by maxyinger — APPROVED** [review]

**Review by BrittanyIRL — APPROVED** [review]
Links present as you specify, 👍

File: assets/src/dashboard/templates/raw/fashion.json [hunk 1]
```diff
@@ -4570,7 +4570,12 @@
           "basedOn": "6f7115f5-66d3-47b1-ad9c-ef16c1aabdf3",
           "id": "f8d34cbd-6ead-46c3-aef2-1113b01df08e",
           "x": 135,
-          "y": 114
+          "y": 114,
+          "link": {
+            "url": "http://example.com/",
```
Comment (swissspidy) [inline assets/src/dashboard/templates/raw/fashion.json]: Nit: let's use `https://` everywhere

Comment (mariano-formidable) [reply assets/src/dashboard/templates/raw/fashion.json]: Done 👍

**Comment by mariano-formidable** [comment]
@googlebot I consent

**Pull request merged**